### ISSUE TITLE
Feature suggestions

### ISSUE THREAD

**Issue by Surendrajat** [arsvechkarev/Vault#52]
Thanks for this app. Finally, a clean and snappy KeePass compatible password manager. However, there are some features I couldn't find coming from Bitwarden. Listing all of them here for convenience.

Important ones:
1. Password Autofill
2. URI rules (multiple, regex & match detection)
3. Trash

Nice to haves:
1. Password History
2. TOTP

**Comment by arsvechkarev** [comment]
Hi! Thanks for suggestions, and glad you liked the app). Autofill, trash, history - those are important features, I will definitely try to implement them in the nearest future. TOTP - nice to have, but there are already good apps specifically for that, like Aegis. Maybe I will work on that once other stuff is done, but it will take a while because I have limited resources. And I haven't quite understood what you mean by "URI rules", can you explain it in detail or point to an app that has it implemented?

**Comment by Surendrajat** [comment]
Hi. Thanks for the reply. Glad to know these features are already on the roadmap. Sure, Aegis works beautifully for TOTP so it can be implemented if and when you get time. About URI rules, it's essential for Autofill and can be considered a part of this. Anyway linking the bitwarden docs for the same here. https://bitwarden.com/help/uri-match-detection/
